api/v1beta1/baremetalset_types.go

```diff
@@ -28,7 +28,7 @@ type BaremetalSetSpec struct {
 	// Replicas The number of baremetalhosts to attempt to aquire
 	// +kubebuilder:validation:Minimum=0
 	Replicas int `json:"replicas,omitempty"`
-	// Remote URL pointing to desired RHEL qcow2 image (TODO: currently gzipped only) for the set
+	// Remote URL pointing to desired RHEL qcow2 image (gzipped format is required)
```

Comment:
Gzipped format is actually not required.  We support compressed and uncompressed now

Reply:
oh right. Lets drop it. Just trying to make this look better

Reply:
@abays fixed in the PR below